package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@wmde/wikibase-datamodel-types",
+	"version": "0.0.1",
+	"description": "TypeScript definitions for the Wikibase data model expressed as flat JS objects",
+	"main": "",
+	"scripts": {
+		"test": "npm run lint && tsc",
+		"lint": "eslint index.d.ts wikibase-datamodel-tests.ts",
+		"fix": "npm run lint -- --fix"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/wmde/WikibaseDataModelTypes.git"
+	},
+	"author": "Wikibase Developers",
```

Comment:
Do we have an official author name we can use?
I've seen "The Wikidata team" used before.

Reply:
Good point, “The Wikidata team” is what all our extensions use. I’ll change it.